test/gtest/nonpack_conv3d_fwd.hpp

```diff
@@ -44,46 +44,58 @@ struct NonPackTestCase : Conv3DTestCase
     size_t o2;
     size_t o3;
     size_t o4;
+    float alpha;
+    float beta;
     std::vector<size_t> GetInputStrides() { return {i0, i1, i2, i3, i4}; }
     std::vector<size_t> GetWeightStrides() { return {w0, w1, w2, w3, w4}; }
     std::vector<size_t> GetOutputStrides() { return {o0, o1, o2, o3, o4}; }
+    float GetAlpha() { return alpha; }
+    float GetBeta() { return beta; }
 };
 
 template <>
 std::vector<NonPackTestCase> ConvTestConfigs()
-{ // g    n   c   d    h   w   k   z  y  x pad_x pad_y pad_z stri_x stri_y stri_z dia_x dia_y dia_z
-    return {{{1, 4, 16, 4, 9, 16, 16, 3, 3, 3, 1, 1, 1, 1, 1, 1, 1, 1, 1, miopenConvolution},
-             10240,
-             1,
-             2560,
-             160,
-             16,
-             432,
-             1,
-             144,
-             48,
-             16,
-             9216,
-             1,
-             2304,
-             256,
-             16},
-            {{1, 1, 64, 3, 16, 16, 128, 3, 3, 3, 1, 1, 1, 1, 1, 1, 1, 1, 1, miopenConvolution},
-             65536,
-             1,
-             24000,
-             2048,
-             64,
-             1728,
-             1,
-             576,
-             192,
-             64,
-             98304,
-             1,
-             32768,
-             2048,
-             128}};
+{ // g  n  c  d  h  w  k  z  y  x  pad_x pad_y pad_z stri_x stri_y stri_z dia_x dia_y dia_z
+    return {{
+                {1, 4, 16, 4, 9, 16, 16, 3, 3, 3, 1, 1, 1, 1, 1, 1, 1, 1, 1, miopenConvolution},
+                10240,
+                1,
+                2560,
+                160,
+                16,
+                432,
+                1,
+                144,
+                48,
+                16,
+                9216,
+                1,
+                2304,
+                256,
+                16,
+                1.0, // alpha
+                0.0  // beta
+            },
+            {
+                {1, 1, 64, 3, 16, 16, 128, 3, 3, 3, 1, 1, 1, 1, 1, 1, 1, 1, 1, miopenConvolution},
+                65536,
+                1,
+                24000,
+                2048,
+                64,
+                1728,
+                1,
+                576,
+                192,
+                64,
+                98304,
+                1,
+                32768,
+                2048,
+                128,
+                1.0, // alpha
```

Comment:
let's not add `alpha/beta` here. There's a better way using `Combine`. See: https://github.com/ROCm/MIOpen/blob/develop/test/gtest/fused_conv_bias_res_add_activ.cpp#L229-L235